src/common/actions/snap-builds.js

```diff
@@ -17,169 +21,52 @@ export function fetchBuildsError(error) {
-    'score': null,
-    'self_link': 'https://api.launchpad.net/devel/~cjwatson/+snap/godd-test-2/+build/1063',
-    'date_started': '2016-06-02T07:54:02.769338+00:00',
-    'resource_type_link': 'https://api.launchpad.net/devel/#snap_build',
-    'build_log_url': 'https://launchpad.net/~cjwatson/+snap/godd-test-2/+build/1063/+files/buildlog_snap_ubuntu_xenial_amd64_godd-test-2_BUILDING.txt.gz',
-    'pocket': 'Updates',
-    'dependencies': null,
-    'date_first_dispatched': '2016-06-02T07:54:02.769338+00:00',
-    'distribution_link': 'https://api.launchpad.net/devel/ubuntu',
-    'distro_series_link': 'https://api.launchpad.net/devel/ubuntu/xenial',
-    'web_link': 'https://launchpad.net/~cjwatson/+snap/godd-test-2/+build/1063',
-    'datecreated': '2016-06-01T13:44:04.646067+00:00',
-    'archive_link': 'https://api.launchpad.net/devel/ubuntu/+archive/primary',
-    'arch_tag': 'amd64',
-    'upload_log_url': null
-  }, {
-    'can_be_rescored': false,
-    'builder_link': 'https://api.launchpad.net/devel/builders/lcy01-05',
-    'datebuilt': '2016-06-02T07:56:07.748129+00:00',
-    'distro_arch_series_link': 'https://api.launchpad.net/devel/ubuntu/xenial/i386',
-    'snap_link': 'https://api.launchpad.net/devel/~cjwatson/+snap/godd-test-2',
-    'duration': '0:01:50.046060',
-    'can_be_cancelled': false,
-    'title': 'i386 build of godd-test-2 snap package in ubuntu xenial-updates',
-    'buildstate': 'Successfully built',
-    'requester_link': 'https://api.launchpad.net/devel/~cjwatson',
-    'http_etag': '\'c34118394abae39240e95f2bc620f6f96a379a68-264ca7acd85945668d342842b2039d75d7b673ff\'',
-    'score': null,
-    'self_link': 'https://api.launchpad.net/devel/~cjwatson/+snap/godd-test-2/+build/1064',
-    'date_started': '2016-06-02T07:54:17.702069+00:00',
-    'resource_type_link': 'https://api.launchpad.net/devel/#snap_build',
-    'build_log_url': 'https://launchpad.net/~cjwatson/+snap/godd-test-2/+build/1064/+files/buildlog_snap_ubuntu_xenial_i386_godd-test-2_BUILDING.txt.gz',
-    'pocket': 'Updates',
-    'dependencies': null,
-    'date_first_dispatched': '2016-06-02T07:54:17.702069+00:00',
-    'distribution_link': 'https://api.launchpad.net/devel/ubuntu',
-    'distro_series_link': 'https://api.launchpad.net/devel/ubuntu/xenial',
-    'web_link': 'https://launchpad.net/~cjwatson/+snap/godd-test-2/+build/1064',
-    'datecreated': '2016-06-01T13:44:04.646067+00:00',
-    'archive_link': 'https://api.launchpad.net/devel/ubuntu/+archive/primary',
-    'arch_tag': 'i386',
-    'upload_log_url': null
-  }],
-  'resource_type_link': 'https://api.launchpad.net/devel/#snap_build-page-resource'
-};
+export function fetchBuilds(snapLink) {
+  return (dispatch) => {
+    if (snapLink) {
+      dispatch({
+        type: FETCH_BUILDS
+      });
+
+      snapLink = encodeURIComponent(snapLink);
+      const url = `${BASE_URL}/api/launchpad/builds?snap_link=${snapLink}`;
```

Comment:
I know the server handler is already in place, but I'd be inclined to call the query parameter here just `snap` rather than `snap_link`.  The Launchpad API sprinkles `_link` suffixes everywhere, but they're a bit cumbersome to deal with directly, and they're normally intended to be wrapped automatically by language bindings (even though our JS binding doesn't yet do so; the Python binding does).  It's probably best not to replicate that here.